assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -294,8 +294,10 @@ che.openshift.liveness.probe.delay=300
 che.openshift.liveness.probe.timeout=1
 che.openshift.workspaces.pvc.name=claim-che-workspace
 che.openshift.workspaces.pvc.quantity=10Gi
-che.openshift.workspace.cpu.limit=1
-# Override memory limit used for openshift workspaces. String, e.g. 1300Mi
+# The amount of memory required for a workspace container to run e.g. 512Mi
```

Comment:
WDYT about leaving the `cpu.limit` in place? It does make things longer and less readable, but in terms of a general solution it would be useful to be able to specify this value, even if it doesn't apply to our specific case. It would be handy e.g. if running on minishift with a simple quota in place.